Nitric.Test/Api/Document/DocumentsTest.cs

```diff
@@ -242,6 +265,31 @@ public void TestSetWithSortedDictionary()
 
             dc.Verify(t => t.Set(It.IsAny<DocumentSetRequest>(), null, null, It.IsAny<System.Threading.CancellationToken>()), Times.Once);
         }
+        [Fact]
+        public void TestSetToNonExistentDocument()
```

Comment:
Might want to test for a different error here. Setting to a non-existing document is perfectly valid, so it's an unrealistic test.

Reply:
I've changed this to testing a permission denied error.